packages/tools/devtools/devtools-browser-extension/src/popup/PopupView.tsx

```diff
@@ -0,0 +1,104 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+import {
+	DevtoolsFeatures,
+	GetDevtoolsFeatures,
+	type ISourcedDevtoolsMessage,
+	type InboundHandlers,
+	handleIncomingMessage,
+} from "@fluidframework/devtools-core";
+import { type BackgroundConnection } from "../BackgroundConnection";
+
+const queryTimeoutInMilliseconds = 2000; // 2 seconds
+
+const loggingContext = "FLUID_DEVTOOLS(Popup)";
+
+/**
+ * Message sent to the webpage to query for the supported set of Devtools features.
+ */
+const getSupportedFeaturesMessage = GetDevtoolsFeatures.createMessage();
+
+/**
+ * Props containing critical BG service connection used for message passing with webpage
+ */
+export interface PopupViewProps {
+	backgroundServiceConnection: BackgroundConnection;
+}
+
+/**
+ * Component that renders when you click extension
+ * @returns popup component
+ */
+export function PopupView(props: PopupViewProps): React.ReactElement {
+	const { backgroundServiceConnection } = props;
+
+	// Set of features supported by the Devtools. True is devtools found, false not found and undefined means still looking
+	const [foundDevtools, setFoundDevtools] = React.useState<boolean | undefined>(undefined);
+
+	React.useEffect(() => {
+		/**
+		 * Handlers for inbound messages related to the registry.
+		 */
```

Comment:
I think the registry is terminology we stopped using.
```suggestion
		/**
		 * Handlers for inbound messages.
		 */
```